src/cascadia/TerminalControl/ControlInteractivity.cpp

```diff
@@ -334,14 +334,17 @@ namespace winrt::Microsoft::Terminal::Control::implementation
                                             const ::Microsoft::Terminal::Core::ControlKeyStates modifiers,
                                             const bool focused,
                                             const Core::Point pixelPosition,
-                                            const bool pointerPressedInBounds)
+                                            const bool pointerPressedInBounds,
+                                            bool& suppressFurtherHandling)

```

Comment:
If we change this to a return value it'd probably be worth adding a comment that "return true" means "suppress further handlings" just in case this helps someone in the future.